chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/developer/attestation/DeveloperAttestationReportDataCollection.java

```diff
@@ -345,9 +346,8 @@ private String getApiValidation(Developer developer, Logger logger) {
     private Map<Pair<Long, Long>, Boolean> getDeveloperAcbMapping(Developer developer, Logger logger) {
         Map<Pair<Long, Long>, Boolean> developerAcbMap = new HashMap<Pair<Long, Long>, Boolean>();
 
-        getListingDataForDeveloper(developer, logger).stream()
-                .filter(listing -> isListingActiveDuringPeriod(listing, attestationPeriodService.getMostRecentPastAttestationPeriod()))
-                .forEach(listing -> developerAcbMap.put(Pair.of(developer.getId(), listing.getCertificationBody().getId()), true));
+        attestationCertificationBodyService.getAssociatedCertificationBodies(developer.getId()).stream()
```

Comment:
What's happening here? Why change commented out code?

Reply:
This will be the "new" code for how we determine what ACBs are associated to a developer for the most recent past Attestation Period.  Just left it in here so we're sure it gets changed, because the original code would still work.